health/micro-ui/web/micro-ui-internals/packages/modules/campaign-manager/src/components/CampaignDocumentsPreview.js

```
@@ -37,8 +37,8 @@ function CampaignDocumentsPreview({ documents = [], svgStyles = {}, isUserGenera
           documents?.map(
             (document, index) =>
               (document?.id || document?.filestoreId) && (
-                <div key={index} style={{ marginRight: "1rem" }}>
-                  <div onClick={() => setShowPreview(true)}>
+                <div key={index} style={{ marginRight: "1rem" , display: "flex"  }}>
+                  <div onClick={() => setShowPreview(true)} className="campaign-preview">
```

Biome

[error] Enforce to have the onClick mouse event with the onKeyUp, the onKeyDown, or the onKeyPress keyboard event.

Actions triggered using mouse events should have corresponding keyboard events to account for keyboard-only navigation.

(lint/a11y/useKeyWithClickEvents)